docs/docs/providers/cognito.md

```diff
@@ -46,4 +46,8 @@ The issuer is a URL, that looks like this: `https://cognito-idp.{region}.amazona
 Make sure you select all the appropriate client settings or the OAuth flow will not work.
 :::
 
-![cognito](https://user-images.githubusercontent.com/7902980/83951604-cd096e80-a832-11ea-8bd2-c496ec9a16cb.PNG)
+:::tip
+Before you can set these settings, you must [set up an Amazon Cognito hosted domain](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-assign-domain.html)
+:::
+
+![cognito](https://private-user-images.githubusercontent.com/11452899/297044112-41954090-1169-4c1b-9f09-ae7ee80a7cbd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyMTI1ODksIm5iZiI6MTcxMzIxMjI4OSwicGF0aCI6Ii8xMTQ1Mjg5OS8yOTcwNDQxMTItNDE5NTQwOTAtMTE2OS00YzFiLTlmMDktYWU3ZWU4MGE3Y2JkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE1VDIwMTgwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFmY2YzZjExOGNkYTdiY2JjZjc4ZmRjZTIyYWI5NzU5ZmRiNzdmMjEwOTkzMWY5MzRjODc0Y2NjODgwZjMzNDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ic0p91xFA7YGbdf2wP5OfzpjkI3p70uGDmAU8a5ir1M)
```

Comment:
Unfortunately we can't view that image. It's private to you. Github changed their user image hosting setup to avoid folks like us using them for free image hosting :sweat_smile: 

Is this screenshot crucial? Maybe the text description is enough?

Reply:
@ndom91 I'll remove the screenshot